src/app/(BottomTabNavigation)/AllCases/utils.ts

```diff
@@ -36,18 +43,39 @@ export async function fetchAllCases(userUid: UserUid): Promise<Case[]> {
  * @returns readable date string
  */
 export function formatDate(dateObject: Date) {
-  const date = new Date(dateObject).toDateString().split(' ');
-  return `${date[1]} ${date[2]}, ${date[3]}`;
+  const userTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  const date = new Date(
+    dateObject.toLocaleString('en-US', {
+      timeZone: userTimeZone,
+    }),
+  );
+  const months = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+  ];
+  const monthName = months[date.getMonth()];
+  return `${monthName} ${date.getDate()}, ${date.getFullYear()}`;
 }
 
 export function getStatusColor(status: string) {
+  if (!StatusOptions.includes(status)) {
+    throw new Error(`Invalid status: ${status}`);
+  }
+
   if (
-    status === 'In Progress' ||
-    status === 'New Case' ||
-    status === 'Settled' ||
-    status === 'Appeal' ||
-    status === 'Payment Processing' ||
-    status === 'Payment Distributed'
+    Object.values(GreenStatusOptions).includes(
+      status as unknown as GreenStatusOptions,
+    )
```

Comment:
hello arfa,

can you possibly remove the `as unknown as GreenStatusOptions` and subsequent occurences? It just seems a little bit redundant. functionally it won't do anything different I think

Warm regards,
Philip

Reply:
done